java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6813.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Avoid field dependency injection",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6813",
+  "sqKey": "S6813",
+  "scope": "All",
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "MEDIUM",
+      "RELIABILITY": "MEDIUM"
```

Comment:
Do we need the `SECURITY` code impact set here, or is it okay to leave it out?

Reply:
As far as I understand it, we only need the impacts that a rule really has. So it is okay to remove the `SECURITY` impact, if the issue doesn't touch security. At least many other migrated rules in RSPEC only have 1 of these defined.